packages/flutter_tools/pubspec.yaml

```diff
@@ -68,7 +68,7 @@ dependencies:
   vm_service: 14.2.0
 
   standard_message_codec: 0.0.1+4
-  frontend_server_client: 3.2.0 # exact pin because of https://github.com/flutter/flutter/issues/146164
+  frontend_server_client: 4.0.0
```

Comment:
This no longer needs to be exactly pinned, can you actually just delete this line and then re-run `flutter update-packages --force-upgrade`? It should add this same version but with the `# THIS LINE IS AUTOGENERATED...` comment

Reply:
Ah, right. Done.